README.md

```diff
@@ -80,22 +80,16 @@ pkg-config --cflags --libs dbus-1
 The minimal setup to develop your package is to create a virtual environment and install the package and its runtime requirements:
 
 ```bash
-# Your Python installation may instead be available under `py` or `python`.
-python3 -m venv venv
-# In Windows PowerShell run this instead: `.\venv\Scripts\Activate.ps1`
-# In Windows CMD run this instead: `.\venv\scripts\activate.bat`
-source venv/bin/activate
-pip install --upgrade pip tox setuptools
-pip install -e '.[dev]'
+pipenv install --dev
 ```
 
 ### Run usdb_syncer
 
-The package has a defined entry point for the GUI. Simply type in `usdb_syncer` in your terminal. Make sure that your venv is activated.
+The package has a defined entry point for the GUI. Simply type in `pipenv run usdb_syncer` in your terminal.
```

Comment:
Wouldn't it be easier to activate the venv with `pipenv shell`? Then you don't have to change all the commands.
`pipenv run usdb_syncer` is also quite cumbersome to type, as there is no atuo completion. In the venv I can just type `u` + <kbd>Tab</kbd> or so.

Reply:
actially both is possible. Its my personal preference to invole the `pipenv run`-syntax, because I do not need to struggle with beeing inside the venv or not.
If you prefer, I can revert the text changes and add `pipenv shell` to end of the code block

Reply:
Thanks, I would prefer that. Then we can keep using our entry points just like before.